tests/lean/run/rewrites.lean

```diff
@@ -0,0 +1,126 @@
+attribute [refl] Eq.refl
+
+private axiom test_sorry : ∀ {α}, α
+
+-- To see the (sorted) list of lemmas that `rw?` will try rewriting by, use:
+-- set_option trace.Tactic.rewrites.lemmas true
+
+/--
+info: Try this: rw [@List.map_append]
```

Comment:
Ah --- I was just noticing this while otherwise happily using `rw_search` to repair some proofs in Mathlib. There is a pretty unnecessary `@` at the front of each lemma name here, because of the way we've going from `Name` to `Expr`. Do you see a way to avoid having this initial `@`?

Reply:
Blind guess: use https://leanprover-community.github.io/mathlib4_docs/Std/Lean/Delaborator.html#Lean.ppConst in the right place?

Reply:
@nomeata Thanks.  I just tested it and it seems to work by upstreaming and inserting into `Tactic.TryThis.addRewrite suggestion`.   I'll make a separate PR for it since it's largely orthogonal to this work.